cookbook/recipes/mysql.rb

```diff
@@ -14,10 +28,18 @@
   action :create
 end
 
+file '/etc/mysql-default/root_password' do
```

Comment:
It actually might make more sense to do this in `/root/.my.cnf` but I didn't want to create any potential conflicts with existing setups.